website/www/site/i18n/home/quotes/en.yaml

```diff
@@ -11,4 +11,4 @@
 # limitations under the License.
 
 - id: home-quotes-title
-  translation: "They tried it out"
\ No newline at end of file
+  translation: "Case Studies Powered by Apache Beam"
```

Comment:
"Case studies powered by Apache Beam"

Reply:
As discussed on today's call, we keep title case everywhere.